bin/setup

```diff
@@ -7,19 +7,19 @@ mkdir -p build/
 if [ ! -d build/go_bootstrap ]; then
   GOOS=$(uname | tr '[:upper:]' '[:lower:]')
   GOARCH=amd64
-  BOOTSTRAP_URL=https://storage.googleapis.com/golang/go1.6.3.$GOOS-$GOARCH.tar.gz
+  BOOTSTRAP_URL=https://storage.googleapis.com/golang/go1.7.5.$GOOS-$GOARCH.tar.gz
```

Comment:
Good question! I think it was required to use 1.4 to bootstrap 1.5, and we've just carried that forward. Maybe we can try using the latest next time!